test/k8s/hubble.go

```diff
@@ -282,7 +282,7 @@ var _ = Describe("K8sAgentHubbleTest", func() {
 				"--last 1 --type trace:from-endpoint --from-pod %s/%s --to-fqdn %s",
 				namespaceForTest, appPods[helpers.App2], fqdnTarget))
 			Expect(flows).To(HaveLen(1))
-			Expect(flows[0].Destination.Identity).To(BeNumerically(">=", identity.MinimalNumericIdentity))
+			Expect(flows[0].GetFlow().Destination.Identity).To(BeNumerically(">=", identity.MinimalNumericIdentity))
```

Comment:
nit: Since we're dereferencing anyway we could
```suggestion
			Expect(flows[0].Flow.Destination.Identity).To(BeNumerically(">=", identity.MinimalNumericIdentity))
```
We'll get the same error in the failure case, and in the context of tests accessing the struct members directly avoids hiding where the `nil` value is in the chain, if that make sense.

Reply:
`.Flow` doesn't work here unfortunately, since `GetFlow()` does some type casting to determine the `oneof` response:

https://github.com/cilium/cilium/blob/1203d9e841b6cb10373bb88a0196f1d62fe78466/api/v1/observer/observer.pb.go#L686-L691

Reply:
Ah yes thank you I forgot about that :+1: